h2o-core/src/test/java/water/rapids/ast/prims/reducers/AstTopNTest.java

```diff
@@ -0,0 +1,119 @@
+package water.rapids.ast.prims.reducers;
+
+import hex.DMatrix;
+import hex.SplitFrame;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.DKV;
+import water.Key;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.rapids.Rapids;
+import water.rapids.Val;
+import water.util.FrameUtils;
+import java.util.Random;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+	* Test the AstTopN.java class
+	*/
+public class AstTopNTest extends TestUtil {
+		static Frame _train;    // store training data
```

Comment:
Why do you have such huge space here? It does not match code above at all.

May you pls fix it?